book/getting-started/run-a-wallaroo-application-docker.md

```diff
@@ -0,0 +1,205 @@
+# Run an Wallaroo Application in Docker
```

Comment:
'an' -> 'a'

Reply:
@nisanharamati that's what I had before, then @SeanTAllen suggested:

"Run A Application in Docker" <-- an application not a application

One of you care to make the decision?

Reply:
There are two different cases:
Run **a W**allaroo Application...
Run **An A**pplication...

Reply:
what nisan said. that's the correct grammar.